### PR TITLE
Reply preview styles

### PR DESCRIPTION
**Pull request by liywjl**
## What does this change?
Changes styles of reply previews

## Why?
Parity

## Link to supporting Trello card
https://trello.com/c/k8JhVYit/1382-style-comment-preview-when-replying

## Before
Screenshot 2020-04-02 at 17 03 55

## After
Screenshot 2020-04-02 at 17 14 34

### CODE REVIEW

File: src/components/CommentReplyPreview/CommentReplyPreview.tsx [hunk 1]
```diff
@@ -7,55 +7,70 @@ import { neutral, space, palette } from "@guardian/src-foundations";
 import { Pillar, CommentType } from "../../types";
 
 const commentControlsButton = (pillar: Pillar) => css`
-  font-weight: bold;
+  ${textSans.small()};
   margin-right: ${space[2]}px;
   color: ${palette[pillar][400]};
```
Comment (oliverlloyd) [inline src/components/CommentReplyPreview/CommentReplyPreview.tsx:12]: Any chance of dropping palette?

File: src/components/CommentReplyPreview/CommentReplyPreview.tsx [hunk 2]
```diff
@@ -7,55 +7,70 @@ import { neutral, space, palette } from "@guardian/src-foundations";
...
   border: 0;
 `;
 
 const replyWrapperStyles = css`
+  padding-top: 5px;
```
Comment (oliverlloyd) [inline src/components/CommentReplyPreview/CommentReplyPreview.tsx:17]: Use `space`?

File: src/components/CommentReplyPreview/CommentReplyPreview.tsx [hunk 3]
```diff
@@ -7,55 +7,70 @@ import { neutral, space, palette } from "@guardian/src-foundations";
...
   position: relative;
   display: flex;
   flex-direction: column;
   :before {
     content: "";
     position: absolute;
-    border-right: ${arrowSize}px solid transparent;
-    border-top: ${arrowSize}px solid ${bg};
-    bottom: -${arrowSize - 1}px;
+    border-left: ${arrowSize}px solid #ededed;
```
Comment (oliverlloyd) [inline src/components/CommentReplyPreview/CommentReplyPreview.tsx:58]: No hex

File: src/components/CommentReplyPreview/CommentReplyPreview.tsx [hunk 4]
```diff
@@ -98,30 +113,53 @@ export const CommentReplyPreview = ({
...
+        }}
+      />
+      <div>
+        <span
+          className={cx(
+            hideCommentButtonStyles,
+            commentControlsButtonStyles,
+            previewHideCommentButtonStyles
+          )}
+          onClick={() => setDisplayReplyComment(!displayReplyComment)}
```
Comment (oliverlloyd) [inline src/components/CommentReplyPreview/CommentReplyPreview.tsx:158]: We should not be using `onClick` on spans as this is not accessible

File: src/components/CommentReplyPreview/CommentReplyPreview.tsx [hunk 5]
```diff
@@ -98,30 +113,53 @@ export const CommentReplyPreview = ({
...
+        {commentBeingRepliedTo.date} said:
+      </p>
+      <div
+        className={commentStyles}
+        dangerouslySetInnerHTML={{
+          __html: commentBeingRepliedTo.body || ""
+        }}
+      />
+      <div>
+        <span
```
Comment (oliverlloyd) [inline src/components/CommentReplyPreview/CommentReplyPreview.tsx:152]: This should be a Button from source

File: src/components/CommentReplyPreview/CommentReplyPreview.tsx [hunk 6]
```diff
@@ -98,30 +113,53 @@ export const CommentReplyPreview = ({
...
 };
+
+export const Preview = ({
+  commentBeingRepliedTo,
+  pillar,
+  setDisplayReplyComment,
+  displayReplyComment
+}: {
+  commentBeingRepliedTo: CommentType;
+  pillar: Pillar;
```
Comment (oliverlloyd) [inline src/components/CommentReplyPreview/CommentReplyPreview.tsx:134]: pillar isn't needed here

File: src/components/CommentReplyPreview/CommentReplyPreview.tsx [hunk 7]
```diff
@@ -98,30 +113,53 @@ export const CommentReplyPreview = ({
...
+  setDisplayReplyComment: (displayReplyComment: boolean) => void;
+  displayReplyComment: boolean;
+}) => {
+  const commentControlsButtonStyles = commentControlsButton(pillar);
```
Comment (oliverlloyd) [inline src/components/CommentReplyPreview/CommentReplyPreview.tsx:138]: Button should not coloured by pillar

Screenshot 2020-04-02 at 18 38 46

File: src/components/CommentReplyPreview/CommentReplyPreview.tsx [hunk 8]
```diff
@@ -7,55 +7,70 @@ import { neutral, space, palette } from "@guardian/src-foundations";
...
+  padding-bottom: 5px;
   display: flex;
   flex-direction: row;
-  padding: 5px;
+  align-items: center;
 `;
 const replyArrowStyles = css`
   fill: grey;
-  padding: 5px;
+  padding-top: 2px; /* TODO: find a better solution */
```
Comment (oliverlloyd) [inline src/components/CommentReplyPreview/CommentReplyPreview.tsx:25]: `find a better solution` - it's more usefule if the comment explains what the problem is with using padding and why it had to be used anyway

File: src/components/CommentReplyPreview/CommentReplyPreview.tsx (hunk 5 above)
Comment (liywjl) [reply src/components/CommentReplyPreview/CommentReplyPreview.tsx]: I tried but it assumes the background is white, so we get a strange  effect
Screenshot 2020-04-02 at 22 12 16

I decided to keep with a basic HTML button instead. We should likely raise the issue in src-foundations

File: src/components/CommentReplyPreview/CommentReplyPreview.tsx [hunk 9]
```diff
@@ -2,66 +2,86 @@ import React, { useState } from "react";
...
   flex-direction: row;
-  padding: 5px;
+  align-items: center;
 `;
 const replyArrowStyles = css`
   fill: grey;
-  padding: 5px;
+  /*
+    In order to get teh arrow SVG alinged correctly with the text reply text
+    we need to add 2px padding
```
Comment (oliverlloyd) [inline src/components/CommentReplyPreview/CommentReplyPreview.tsx:27]: ❤️

**Pull request merged**